script/client/spring/application.yml

```diff
@@ -69,7 +69,7 @@ seata:
       namespace:
       serverAddr: 127.0.0.1:8848
       group: SEATA_GROUP
-      userName: ""
+      username: ""
```

Comment:
Please unify the format of it wherever it's used.like NacosRegistryServiceImp.

Reply:
checked